org.eclipse.draw2d.tests/META-INF/MANIFEST.MF

```diff
@@ -2,14 +2,17 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Plugin.name
 Bundle-SymbolicName: org.eclipse.draw2d.tests
-Bundle-Version: 3.13.100.qualifier
+Bundle-Version: 3.14.0.qualifier
+Import-Package: org.slf4j
 Bundle-Vendor: %Plugin.providerName
 Bundle-Localization: plugin
 Require-Bundle: org.eclipse.core.runtime;bundle-version="[3.2.0,4.0.0)",
  org.eclipse.core.resources;bundle-version="[3.2.0,4.0.0)",
  org.eclipse.draw2d;bundle-version="[3.16.0,4.0.0)",
+ org.eclipse.draw2d.examples;bundle-version="3.15.400",

```

Comment:
shouldn't this be 3.16.0 and should we really be so explicit?

Reply:
> shouldn't this be 3.16.0 and should we really be so explicit?

I think I've raised the version number after adding it as a dependency. But you're right. For dependencies within the same project, the version constraint is probably not needed...